src/Elm/Declare.elm

```diff
@@ -225,9 +351,10 @@ function :
     -> List ( String, Maybe Elm.Annotation.Annotation )
     -> (List Expression -> Expression)
     ->
-        { declaration : Declaration
-        , call : List Expression -> Expression
-        , callFrom : List String -> List Expression -> Expression
+        { declaration : Elm.Declaration
```

Comment:
What if you instead accept a `Fn tipe` and return a `Decl tipe`? Or something in that vein